backend/apps/slack/models/conversation.py

```diff
@@ -0,0 +1,179 @@
+"""Slack app conversation model."""
+
+import logging
+from datetime import datetime, timezone
+
+from django.db import models
+
+from apps.common.models import BulkSaveModel, TimestampedModel
+
+logger = logging.getLogger(__name__)
+
+
+class Conversation(BulkSaveModel, TimestampedModel):
+    """conversation model."""
+
+    class Meta:
+        db_table = "slack_conversations"
+        verbose_name_plural = "Conversations"
+
+    created_at = models.DateTimeField(verbose_name="Created At", blank=True, null=True)
+    creator_id = models.CharField(verbose_name="Creator ID", max_length=255)
+    entity_id = models.CharField(verbose_name="Entity ID", max_length=255, unique=True)
+    is_archived = models.BooleanField(verbose_name="Is Archived", default=False)
+    is_general = models.BooleanField(verbose_name="Is General", default=False)
+    is_private = models.BooleanField(verbose_name="Is Private", default=False)
+    name = models.CharField(verbose_name="Name", max_length=255)
+    purpose = models.TextField(verbose_name="Purpose", blank=True, default="")
+    topic = models.TextField(verbose_name="Topic", blank=True, default="")
+
+    def __str__(self):
+        """Return a string representation of the conversation."""
+        return self.name
+
+    @classmethod
+    def prepare_from_slack_data(cls, conversations):
```

Comment:
Could you structure and implement is similar to other `from_*` and `update_*` methods we have for other entities (see github app for an example)?